ephios/user_management/templates/user_management/userprofile_form.html

```diff
@@ -21,10 +21,67 @@ <h1>{% trans "Add new user" %}</h1>
     </div>
     <form method="post" class="form">
         {% csrf_token %}
-        {% bootstrap_form form %}
+        {% bootstrap_form userprofile_form %}
+        {% load formset_tags %}
+
+        <div class="formset" data-formset data-formset-prefix="{{ qualification_formset.prefix }}">
+            {{ qualification_formset.management_form }}
+
+            <div class="card mb-2">
+                <div class="card-body">
+                    <h3 class="card-title">{% translate "Qualifications" %}</h3>
+                    <ul class="list-group list-group-flush" data-formset-body>
+                        <!-- New forms will be inserted in here -->
+                        {% for form in qualification_formset %}
+                            <div class="list-group-item clearfix" data-formset-form>
+                                {{ form.id }}
+                                <button class="btn btn-link float-right pt-4" type="button"
+                                        data-formset-delete-button><span
+                                        class="fas fa-trash-alt"></span> {% trans "Delete" %}</button>
+                                <div class="form-row">
+                                    <div class="col">
+                                        {% bootstrap_field form.qualification %}
```

Comment:
have you tried this?
```suggestion
                                        {% bootstrap_field form.qualification layout="horizontal" %}
```

Reply:
(just so it takes up less vertical space)

Reply:
good idea, makes it even smaller